xla/xla.proto

```diff
@@ -819,6 +819,8 @@ message DebugOptions {
   // Whether to use multiple compute streams to run windowed einsum.
   bool xla_gpu_multi_streamed_windowed_einsum = 280;
 
+  bool xla_gpu_experimental_stream_annotation = 341;
```

Comment:
Nit: add a comment describing what this does.

Reply:
Fixed.